src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -141,7 +145,12 @@ public String runKubectlCommand(String command, ImmutableList<String> args)
       throw e;
     } finally {
       for (String tempFile : tempFiles) {
-        getWorkspace().child(tempFile).delete();
+        try {
+          getWorkspace().child(tempFile).delete();
+        } catch (Exception ee) {
```

Comment:
Is it possible to make this more explicit to know whether this exception happened because the parent dir was deleted or because something unexpected happened which we should probably log at a minimum

Reply:
Im not sure how we'd accomplish telling the difference, as both as surfaces as IOExceptions. That said, agree we should be logging it so that we can leave a log trail in-case it becomes an issue.